3-extra/1-choose-your-own-adventure.js

```diff
@@ -53,6 +57,15 @@ let game = {
     //
     // Hint: the room objects have north/east/south/west methods which return
     // a new room object that is in the relevant direction.
+
+    if (
+      (direction === "north" || "east" || "south" || "west") &&
+      rooms[this.currentRoom.name][direction]()
```

Comment:
works! you could store `rooms[this.currentRoom.name][direction]()` in a variable to save typing it twice